### DISCUSSION TITLE
Razer keypad support?

### DISCUSSION THREAD

**Discussion by nicodemus144** [AntiMicroX/antimicrox#402] [Q&A] [answered]
hey all! i love the work you're doing with this app.

it there any possibility for Razer keypad support? Like the Tartarus and Orbweaver? i've been using [KBMaster](https://bitbucket.org/monkygames/keyboarding-master/src/master/) but it hasn't seen any updates in 6 years.

I'd heard that ppl were able to use this app for the Azeron keypad, so I figured I'd ask and see if this had been considered.

Thanks!

**Comment by pktiuk** [comment, marked as answer]
There is no list of supported hardware.  
AntiMicroX is not targeted for any specific gamepads or accessories. It uses Simple DirectMedia Libraries and everything supported by them is supported by AntiMicroX (but nothing more).  
If they don't support it, then AntiMicroX won't support it.

**Comment by nicodemus144** [comment]
got it, thanks!
